### ISSUE TITLE
Ability to save/load Manager configurations

### ISSUE THREAD

**Issue by esotericist** [Vswe/ModJam3#34]
I've been in a lot of situations while testing configurations I either needed to set up several managers with the exact same configuration entirely, or I found myself needing to relocate an SFM machine that had a complicated configuration.

It'd be nice if we had a way of storing an SFM's configuration on another object, a data card, schematic, or other item representation of the logic in the machine.

Obviously, it will still be necessary to select inventories/tanks/emitters/receivers for the nodes, but simply having the nodes already in place and (mostly) configured would be a huge reduction in effort.

EDIT: An example of the kind of functionality I'd like is present in the Additional Buildcraft Objects mods' "Gate Settings Duplicator". Right click on a manager, stores the configuration in the item. Shift-right-click on a manager, sets the configuration on the manager to what's in the item.

https://github.com/Flow86/Additional-Buildcraft-Objects-Mod

**Comment by esotericist** [comment]
With the addition of command groups, I think it's more important to be able to save those independently, to build a library of useful toolkits, than the ability to save/load the entirety of a particular manager's configuration.

**Comment by Vswe** [comment]
Remember that you would sill have to re-select all the containers.

**Comment by esotericist** [comment]
Of course. I can't speak for others, but for me the effort of laying out all the nodes, setting the non-container configurations, that's the bulk of the effort. Going back and filling in the containers after the fact is less of a burden, especially with container variables.

Also, if we do get a mechanism for naming containers, that could be even less of a burden, if the saved nodeset remembers container names.
